### ISSUE TITLE
property 'tab' not valid in version 42.0

### ISSUE THREAD

**Issue by wpeter-sfdc** [forcedotcom/sfdx-dreamhouse#167]
Going through the [quickstart trailhead](https://trailhead.salesforce.com/trails/sfdx_get_started/projects/quick-start-salesforce-dx/steps/create-and-test-our-scratch-org) and when I do the command to push source `sfdx force:source:push` I get an error saying:

```
force-app/main/default/applications/DreamHouse.app-meta.xml              Property 'tab' not valid in version 42.0
```

Changing `DreamHouse.app-meta.xml` to use `<tabs>` instead of `<tab>` fixes it

**Comment by vazexqi** [comment]
The source version was changed from 40.0 to 42.0 in https://github.com/forcedotcom/sfdx-dreamhouse/commit/da854249e479dfde56e11e3afe12198b5b2de8ae#diff-0c121a7624d9c2f12aa4c6515c6a2130.

**Comment by dcarroll** [comment]
Thanks Nick

**Comment by vazexqi** [comment]
This is closed via #168
